### PR TITLE
First pass at .NET Core RC2

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+
```
Comment (ivannaranjo) [inline .travis.yml]: Looking at the core-deps Docker image it seems that they might need more packages than this, https://github.com/dotnet/dotnet-docker/blob/master/1.0.0-rc2/core-deps/Dockerfile

Comment (jskeet) [inline .travis.yml]: That was all I needed to get as far as the dotnet tool running, and successfully restoring and building packages. It's only the test that fails.

Comment (ivannaranjo) [inline .travis.yml]: Not that this is the cause of the test failure, but I will assume that those dependencies are required by some parts of the runtime one way or another, so eventually they should be added to the setup.

Comment (jskeet) [inline .travis.yml]: Well they may well be required by ASP.NET but not by just the core. I suspect all of this will change over time, to be honest.

**Comment by jskeet** [comment]
Closing as we have a better PR now.

**Pull request closed without merging**